composer.json

```diff
@@ -21,18 +21,18 @@
   },
   "config": {
     "platform": {
-      "php": "5.6"
+      "php": "7.4"
     }
   },
   "minimum-stability": "stable",
   "require": {
     "php": ">=5.5",
     "ext-json": "*",
     "ext-openssl": "*",
-    "guzzlehttp/guzzle": "^6.4",
```

Comment:
To address the version differences while keeping the PHP version the same, what about doing something like `^6.4|^3.7.0`? (same goes for the UUID package)

Reply:
Nice call. That worked.

Haven't noticed any breaks in `rox-php` on the main Laravel 8 PR and aAll the tests passed when I did `composer install` + `composer test`.